src/backgroundScript.js

```diff
@@ -12,7 +12,7 @@ const backgroundScript = (selectors, colors) => {
 
   const injectSpanElement = (element, randomColor, selector) => {
     const tag = document.createElement("span");
-    tag.innerText = selector;
+    tag.innerHTML = selector;
```

Comment:
This is prone to html with inline js injection, would be better if that was innertext